### ISSUE TITLE
`Too many requests` error to OpenSearch

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/harvest#134]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I did harvest action (2 in parallel), I noticed I received this error:
```
CASE 4: Too Many Requests
 
[ERROR] method [GET], host [[https://search-psa-prod-fec4omrmor3hhu5cflf7zigvtm.us-west-2.es.amazonaws.com:443](https://search-psa-prod-fec4omrmor3hhu5cflf7zigvtm.us-west-2.es.amazonaws.com/)], URI [/registry/_search], status line [HTTP/1.1 429 Too Many Requests]
429 Too Many Requests /registry/_search
```

### 🕵️ Expected behavior

I expected to not receive this error

### 📜 To Reproduce

1. TBD use case from PSA

### 🖥 Environment Info

Linux

### 📚 Version of Software Used

v3.7.6

### 🩺 Test Data / Additional context

TBD

**Comment by alexdunnjpl** [comment]
Retry with expo backoff seems like the obvious solution here

**Comment by jordanpadams** [comment]
Closing per https://github.com/NASA-PDS/harvest/issues/125. Will open new ticket if problem persists
